src/presentation/ui/page/mappings.templ

```diff
@@ -0,0 +1,338 @@
+package page
+
+import (
+	"github.com/speedianet/os/src/domain/dto"
+	"github.com/speedianet/os/src/domain/valueObject"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentMappings "github.com/speedianet/os/src/presentation/ui/component/mappings"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+)
+
+script MappingsIndexLocalState() {
+    document.addEventListener('alpine:init', () => {
+		Alpine.data('mappings', () => ({
```

Comment:
Why did you use spaces over tab here? The UI won't lint the code, so you need to make sure the code is right, it's awfully misaligned. Did you read the PR here on GitHub? Maybe the UI on VSCode isn't showing, but here I can see that a lot of the items are on the wrong indentation, maybe because there is a mix of tabs and spaces.

Reply:
Fixed!